cpp/dolfinx/io/xdmf_utils.h

```diff
@@ -24,6 +26,13 @@
 #include <utility>
 #include <vector>
 
+namespace
```

Comment:
Anonymous namespaces should never appear in header files - should only appear in `.cpp` files.